Formula/telegraf.rb

```diff
@@ -23,7 +24,7 @@ class Telegraf < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", *std_go_args, "-ldflags", "-X main.version=#{version}", "./cmd/telegraf"
+    system "go", "build", *std_go_args(ldflags: "-s -w -X github.com/influxdata/telegraf/internal.Version=#{version}-brew"), "./cmd/telegraf"
```

Comment:
Why suffix the version?

Reply:
To be clear it is not the built version from Influxdata? I was going to add the revision number in this version as well. But your other review comment suggests not to add a revision?

Reply:
If you install it from Homebrew it'll never be the influxdata version. Do Debian etc suffix their Telegraf builds like this?

Reply:
In Debian, you use the official builds: https://docs.influxdata.com/telegraf/v1.21/introduction/installation/